frontend/src/components/Routes.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { Route, withRouter } from 'react-router'
+import Home from './home'
+import Login from './auth'
+import Logout from './auth/Logout'
+import Register from './register'
+import Accounts from './accounts'
+import AccountDetails from './accountdetails'
+import AccountHistory from './accounthistory'
+import Navbar from './navbar'
+import Statusbar from './statusbar'
+import PrivateRoute from './PrivateRoute'
+
+class Routes extends React.Component {
+
+  render() {
+    return (
+      <div>
+        <Navbar />
+        <Statusbar />
+        <Route exact path='/' component={Home} />
+        <PrivateRoute path='/accounts' component={Accounts} auth={this.props.auth.isLoggedIn} />
+        <Route path="/account/:id" component={AccountDetails} />
```

Comment:
I suppose both: /account/:id  and /transactions/:id  should be PrivateRoute too.